.eslintrc.json

```diff
@@ -7,7 +7,13 @@
   "extends": ["next/core-web-vitals", "airbnb-base", "airbnb-typescript/base", "next", "prettier"],
   "rules": {
     "no-unused-vars": [1, { "args": "after-used", "argsIgnorePattern": "^_" }],
-    "prettier/prettier": "error",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ],
+    "no-console": "off",
```

Comment:
We don’t want to turn off this rule. Instead remove console statements over time as the app matures and we don’t need to commit console log statements.

Reply:
I have pushed new changes. I have removed the `"no-console:"off"` rule. Should I also remove the prettier `"endOfLine":"auto"`? Earlier it was throwing [warnings and errors](https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-eslint-delete-cr-prettier-prettier?page=1&tab=scoredesc#tab-top) without it.